src/App.js

```diff
@@ -1,26 +1,47 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+import { HashRouter, Route,  Switch} from "react-router-dom";
+
+
+//pages
+
+import Closet from './containers/closet';
+import Opening from './containers/opening';
+import addItem from './containers/addItem';
+import mixNmatch from './containers/mixNmatch';
+import Calendar from './containers/calendar';
```

Comment:
Very tiny nitpick but should these be capitalized like `AddItem` and `MixNMatch` or is there a good reason not to?